src/store/reducer.js

```diff
@@ -4,12 +4,14 @@ import accountReducer from './account/account.reducer';
 import booksReducer from './books/books.reducer';
 import booksSearchReducer from './search/search.reducer';
 import loginReducer from './login/login.reducer';
+import userReducer from './user/user.reducer';
 
 const rootReducer = combineReducers({
     booksCollection: booksReducer,
     booksSearch: booksSearchReducer,
     account: accountReducer,
     login: loginReducer,
+    user: userReducer,
```

Comment:
I'm happy to see that everything is back to normal here, but I have to say that I'm really confused with the timeline of the committed code